commit 893f15c3ef61f253f02031ddc03863a8e037c118 (refs/changes/46/3646/7)
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2018-06-25T12:12:23-07:00 (1 year, 3 months ago)
    
    joyent/node-manta#343 Want `mjob wait` as alias for `mjob watch`
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 179698b..cfce909 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,8 @@
 
 ## not yet released
 
+- joyent/node-manta#343 Add `mjob wait` as an alias to `mjob watch`.
+
 ## 5.1.1
 
 - joyent/node-manta#315 document mlogin session termination conditions
diff --git a/bin/mjob b/bin/mjob
index b8196de..c3bfe4f 100755
--- a/bin/mjob
+++ b/bin/mjob
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 // -*- mode: js -*-
 /*
- * Copyright 2016 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 var EventEmitter = require('events').EventEmitter;
@@ -1364,6 +1364,7 @@ MJob.prototype.do_watch.help = (
         '{{options}}'
 );
 
+MJob.prototype.do_watch.aliases = ['wait'];
 
 
 //-- Close
diff --git a/man/man1/mjob.1 b/man/man1/mjob.1
index aed8749..14e594a 100644
--- a/man/man1/mjob.1
+++ b/man/man1/mjob.1
@@ -56,8 +56,8 @@ The following commands and options are supported:
 .PP
 Creates a job that executes the commands against keys that will be specified
 via \fB\fCaddinputs\fR\&.  \fB\fCexpression\fR can specify an arbitrary UNIX pipeline, with
-map/reduce \fIphases\fP separated by the \fB\fC^\fR or \fB\fC^^\fR 
-.BR character (s), 
+map/reduce \fIphases\fP separated by the \fB\fC^\fR or \fB\fC^^\fR
+.BR character (s),
 respectively.
 .PP
 For example, to specify a simple \fB\fCgrep | sort | uniq\fR job in Manta, the
@@ -219,7 +219,7 @@ $ mjob get 3ec32136\-b125\-11e2\-8487\-1b418dd6974b
 .RE
 .SS watch JOB
 .PP
-Waits for a given job to reach the \fB\fCdone\fR state.
+Waits for a given job to reach the \fB\fCdone\fR state. Aliases: wait.
 .PP
 .RS
 .nf
